### PR TITLE
Removing Debug Method console logging

### PR DESCRIPTION
**Pull request by igor-ribeiiro**
Removed the following code:

 // enable debugging
        if (isDesktopAppType && isWindowsPlatform) {
            await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);
            if (enableDebugging) {
                console.log(`Enabled debugging for add-in ${manifestInfo.id}. Debug method: ${DebuggingMethod[debuggingMethod]}`);
            }
        }

### CODE REVIEW

File: packages/office-addin-debugging/src/start.ts [hunk 1]
```diff
@@ -290,14 +290,6 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
             await devSettings.ensureLoopbackIsEnabled(name);
         }
 
-        // enable debugging
-        if (isDesktopAppType && isWindowsPlatform) {
-            await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);
```
Comment (TCourtneyOwen) [inline packages/office-addin-debugging/src/start.ts:295]: This should remain:

await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);

Please just remove the logging portion below

File: packages/office-addin-debugging/src/start.ts [hunk 2]
```diff
@@ -290,14 +290,6 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
             await devSettings.ensureLoopbackIsEnabled(name);
         }
 
-        // enable debugging
```
[the remark this answers is not in the document]
Comment (akrantz) [reply packages/office-addin-debugging/src/start.ts:293]: The reason that we're removing the logging of the debugging method is that the proxy debugging method that related to ReactNative is no longer being used, as ReactNative is moving towards direct debugging, so we really only have the one debugging method, so there is no need to log that.

**Review by akrantz — CHANGES_REQUESTED** [review]
Thanks for the pull request! See the comments.

File: packages/office-addin-debugging/src/start.ts (hunk 1 above)
Comment (akrantz) [reply packages/office-addin-debugging/src/start.ts:295]: @TCourtneyOwen See my comments. We want the console.log() line, but just not the "Debug method: {method}" part.

File: packages/office-addin-debugging/src/start.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (TCourtneyOwen) [reply packages/office-addin-debugging/src/start.ts:293]: /Azurepipelines run

File: packages/office-addin-debugging/src/start.ts [hunk 3]
```diff
@@ -293,9 +293,6 @@ export async function startDebugging(manifestPath: string, options: StartDebuggi
         // enable debugging
         if (isDesktopAppType && isWindowsPlatform) {
             await devSettings.enableDebugging(manifestInfo.id, enableDebugging, debuggingMethod, openDevTools);
-            if (enableDebugging) {
```
Comment (akrantz) [inline packages/office-addin-debugging/src/start.ts:296]: Sorry if this has been unclear but we want this console.log message. Just remove the "Debug method: " part at the end.

**Review by akrantz — APPROVED** [review]

**Pull request merged**